### DISCUSSION TITLE
[5.x]: Image editor is not accessible via mouse hover

### DISCUSSION THREAD

**Discussion by davidsprinz** [craftcms/cms#14333] [General]
### Description

In the image slideout of an entry. the image editor is only accessible via the three dots menu right above the image. I'm missing the hover menu.

### Steps to reproduce

1. Double click on an image in an entry / asset field.
2. The slideout for title editing and image details appears.
3. Hover over the image, nothing happens.

### Expected behavior

Expected behaviour from 3.x and 4.x is a hover menu with "preview" and "edit" for quicker access to the image editor.

### Craft CMS version

5.0.0-beta.1

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
It was intentionally removed, but we can consider bringing it back if the button is missed.

**Reply by thupsi** [reply]
+1 from me. The preview and edit image actions are too important to be hidden under the three dots menu. I mean they should also stay in that menu, but the buttons are much more discoverable.

**Comment by brandonkelly** [comment]
Alright, brought those buttons back for the next release (2f16a6d20978002d92a557ced504800dc634f7d6).

**Reply by brandonkelly** [reply]
5.0.0-beta.2 is out now with that change.
